### ISSUE TITLE
Create rule S1082 (`jsx-a11y/mouse-events-have-key-events`): Mouse events should have corresponding keyboard events

### ISSUE THREAD

**Issue by ilia-kebets-sonarsource** [SonarSource/SonarJS#4290]
[plugin rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/mouse-events-have-key-events.md#jsx-a11ymouse-events-have-key-events)

**Comment by ilia-kebets-sonarsource** [comment]
fixed by https://github.com/SonarSource/SonarJS/pull/4301
